Gemfile

```diff
@@ -170,7 +170,7 @@ end
 group :production do
   # Ruby interface to the PostgreSQL RDBMS
   # https://github.com/ged/ruby-pg
-  gem 'pg'
+  gem 'pg', '~> 0.18'
```

Comment:
Why this change in version?

Reply:
Because Travis load the 1.0.0 version automatically and failed with it. After I disabled pg at all, Travis said, that he wants to have version 0.18